app.py

```diff
@@ -20,53 +20,6 @@
     db_url = db_url.replace("postgres://", "postgresql://", 1)
 app.config["SQLALCHEMY_DATABASE_URI"] = db_url
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key = b"Habit Group!"
```

Comment:
Does this need to be moved to .env file? I know it's not anything cryptic, but for "security" purposes. Then you'll use `app.secret_key = os.getenv("SECRET_KEY")`